test/jdk/java/foreign/TestNoForeignUnsafeOverride.java

```diff
@@ -40,6 +40,6 @@
 
     @Test(expectedExceptions = IllegalAccessError.class)
     public void testUnsafeAccess() {
-        MemorySegment.ofNativeRestricted(MemoryAddress.ofLong(42), 10, null, null, null);
+        MemorySegment.ofNativeRestricted();
```

Comment:
Should be:
```suggestion
        MemoryAddress.ofLong(42).asSegmentRestricted(10);
```
Right? To retain the same behaviour?

Reply:
This test is, if I understand correctly, simply testing that you can't calla restricted method if the JDK property has not been set on the command line. It's not important WHICH restricted method is called. At the time the test was written we didn't have the more succint ofNativeRestricted - now we do, so I thought to just use that, for brevity.